service/controller/clusterapi/controller.go

```diff
@@ -45,7 +46,7 @@ func NewController(config ControllerConfig) (*Controller, error) {
 		c := controller.Config{
 			K8sClient: config.K8sClient,
 			Logger:    config.Logger,
-			Name:      "clusterapi-controller",
+			Name:      project.Name() + "-cluster-api-controller",
```

Comment:
Do we want to specify if this is for CPs or TCs?

Reply:
I was thinking we would actually have a control-plane controller as we do not need to differentiate between cluster api and legacy there

Reply:
But we can specify this is for tenant clusters if we think this is necessary

Reply:
I think this is fine, we'll know which one is which, ez